seqlike/SeqLikeAccessor.py

```diff
@@ -61,7 +61,10 @@ def _validate(obj):
         """
 
         # We assume that the 'seqs' column is all SeqLikes
-        assert all([isinstance(x, SeqLike) for x in obj]), "Seq column must contain all SeqLike objects."
+        is_seqlikes = obj.apply(lambda x: isinstance(x, SeqLike))
+        types = obj.apply(lambda x: type(x))
```

Comment:
consider adding a `value_counts()` or `unique()`.  the output below could be really long